doc/api/perf_hooks.md

```diff
@@ -72,23 +72,23 @@ high resolution milliseconds timer. The `utilization` value is the calculated
 Event Loop Utilization (ELU). If bootstrapping has not yet finished, the
 properties have the value of 0.
 
-`util1` and `util2` are optional parameters.
+`utilization1` and `utilization2` are optional parameters.
 
-If `util1` is passed then the delta between the current call's `active` and
-`idle` times are calculated and returned (similar to [`process.hrtime()`][]).
-Likewise the adjusted `utilization` value is calculated.
+If `utilization1` is passed, then the delta between the current call's `active`
+and `idle` times are calculated and returned (similar to
+[`process.hrtime()`][]). Likewise the adjusted `utilization` value is calculated.
 
-If `util1` and `util2` are both passed then the calculation adjustments are
-done between the two arguments. This is a convenience option because unlike
-[`process.hrtime()`][] additional work is done to calculate the ELU.
+If `utilization1` and `utilization2` are both passed, then the delta is
+calculated between the two arguments. This is a convenience option because,
+unlike [`process.hrtime()`][], calculating the ELU is more complex than a
+single subtraction.
 
-ELU is similar to CPU utilization except that it is calculated using high
-precision wall-clock time. It represents the percentage of time the event loop
```

Comment:
I might be missing something here, because the original sentence here didn’t really make sense to me – neither CPU utilization nor event loop utilization use wall-clock time?